velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -388,18 +400,31 @@ TEST_F(StringFunctionsTest, substrVariable) {
     // Checking the null results
     if (expectNullString(i) || expectNullStart(i) || expectNullLength(i)) {
       EXPECT_TRUE(result->isNullAt(i)) << "expected null at " << i;
+      EXPECT_TRUE(substringResult->isNullAt(i)) << "expected null at " << i;
```

Comment:
You can create a lambda function say validateResult does the code in the body of the loop and call it with result and substringResult to avoid the code repetition.

Reply:
done in new commit.